scripts/checkFilesChanged.mts

```diff
@@ -13,19 +14,40 @@ const addedAndModifiedFiles = diff.stdout
   .filter((line) => gitDiffRegex.test(line))
   .map((line) => line.split('\t'));
 
-if (addedAndModifiedFiles.length === 0) {
-  console.log('No schema files have been added or modified since the latest release.');
+const badFiles: string[][] = [];
+
+for (const [action, file] of addedAndModifiedFiles) {
+  if (action === 'D') {
```

Comment:
Where's the condition for allowing new files to be created? Isn't this change supposed to support that too?

Reply:
The regex ignores new files. It takes only the M and D actions (new file is an A action) so new files won't be part of the check. Correct me if I am wrong @CptSchnitz

Reply:
@netanelC true. We check for files that fails the validation and created files are filtered early.